src/components/zoomableImage/index.scss

```diff
@@ -34,7 +34,8 @@
 .zoomable__toggle:focus + .zoomable__container {
   @include focus-border;
 }
-.zoomable__toggle:hover + .zoomable__container {
+.zoomable__toggle:hover + .zoomable__container,
+.zoomable__container:hover {
   @include hover-border;
```

Comment:
I think if we add `cursor: pointer;` to the `hover-border` mixin, the issue with the cursor not being a pointer when hovering some parts around the image would be resolved.

With the change that you made with the selector, we would be good to go.

Reply:
That sounds like a great idea, I will add this fix :)

Reply:
I implemented the fix :)